dvc/stage.py

```diff
@@ -571,8 +571,8 @@ def _fill_stage_outputs(stage, **kwargs):
     def _check_dvc_filename(fname):
         if not Stage.is_valid_filename(fname):
             raise StageFileBadNameError(
-                "bad DVC-file name '{}'. DVC-files should be named"
-                " 'Dvcfile' or have a '.dvc' suffix (e.g. '{}.dvc').".format(
+                "bad DVC-file name '{}'. DVC-files should be named "
```

Comment:
Btw, thanks a lot for fixing the spacing! 🙏

Reply:
Np (: